packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/InAppPurchasePluginTests.swift

```diff
@@ -0,0 +1,849 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Foundation
+import StoreKitTest
+import XCTest
+
+@testable import in_app_purchase_storekit
+
+final class InAppPurchasePluginTests: XCTestCase {
+  var receiptManagerStub: FIAPReceiptManagerStub!
+  var plugin: InAppPurchasePlugin!
+
+  override func setUp() {
+    super.setUp()
+    receiptManagerStub = FIAPReceiptManagerStub()
+    plugin = InAppPurchasePluginStub(receiptManager: receiptManagerStub) { request in
+      DefaultRequestHandler(requestHandler: FIAPRequestHandler(request: request))
+    }
+  }
+
+  override func tearDown() {
+    receiptManagerStub = nil
+    plugin = nil
+    super.tearDown()
+  }
+
+  func testCanMakePayments() {
+    var error: FlutterError?
+    let result = plugin.canMakePaymentsWithError(&error)
+    XCTAssertTrue(result!.boolValue)
```

Comment:
Nit: Try to avoid `!`

Can you look at `XCTUnwrap`: https://www.hackingwithswift.com/example-code/testing/how-to-check-and-unwrap-optionals-in-tests-using-xctunwrap

Reply:
oho I did not know about, thats super helpful